### PR TITLE
ADEN-7149 Turn off a9 if user has opted out of tracking

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/a9.js [hunk 1]
```diff
@@ -169,6 +173,7 @@ define('ext.wikia.adEngine.lookup.a9', [
 		logGroup: logGroup,
 		name: 'a9',
 		call: call,
+		isEnabled: isEnabled,
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/lookup/a9.js]: Where are the usages of the new method?

**Review by nandy-andy — APPROVED** [review]

**Comment by fraszczakszymon** [comment]
What about mobile-wiki?

File: extensions/wikia/AnalyticsEngine/templates/a9.mustache [hunk 2]
```diff
@@ -1,12 +1,9 @@
 <script>
 	require([
-		'ext.wikia.adEngine.geo',
-		'wikia.instantGlobals',
+        'ext.wikia.adEngine.adContext',
```
Comment (fraszczakszymon) [inline extensions/wikia/AnalyticsEngine/templates/a9.mustache]: Fix indent please

**Review by nandy-andy — APPROVED** [review]

**Comment by v3rth** [comment]
mobile-wiki is already implemented in the right way https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/run/mercury.run.js#L91

**Pull request merged**